ci-jobs/templates/ios-e2e-template.yml

```diff
@@ -48,7 +48,9 @@ jobs:
         git checkout f660049bdec9965934973052e6331119a4985096
         brew install idb-companion
         popd
-        python -m pip install --user fb-idb
+        # install idb client
+        brew install python@3.7
```

Comment:
The environment already uses Py3.7 with `UsePythonVersion` in a few lines above, so can here run `pip` install directly, or `python -m pip install ....`?
Or should be via brew for the idb?

Reply:
No, I think there is no reason to install double and I just missed it. Let me fix it, thank you.

Reply:
fixed